kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -53,6 +54,20 @@ public Batch createBatch(BatchRequest request, File resourceFile) {
     return this.getClient().post(API_BASE_PATH, multiPartMap, Batch.class, client.getAuthHeader());
   }
 
+  public Batch createBatchWithSubResources(
+      BatchRequest request, File resourceFile, List<String> subResources) {
```

Comment:
we don't need a different name, just use `createBatch`
```suggestion
  public Batch createBatch(
      BatchRequest request, File mainResource, List<String> extraResources) {
```

Reply:
LGTM. Fixed.